index.bs

```diff
@@ -455,8 +455,9 @@ dictionary IntersectionObserverInit {
 		and vice versa.
 
 		Threshold values must be in the range of [0, 1.0] and represent a
-		percentage of the area as specified by
-		<a>target</a>.{{Element/getBoundingClientRect()}}.
+		percentage of the area of the rectangle produced
+		by running the <a>getBoundingClientRect</a> algorithm
```

Comment:
Ugh, line 52 in the anchors block is wrong/confusing. It should be `type: method; text: getBoundingClientRect(); for: Element`.  What it currently says is nonsense.